src/screens/Wallet/WalletList/WalletListContainer.js

```diff
@@ -3,19 +3,124 @@
  * Licensed under the AGPL Version 3 license.
  */
 
+import React, { PureComponent } from 'react'
 import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import I18n from 'react-native-i18n'
+// import { sectionsSelector } from '../redux/mainWallet/selectors'
+import {
+  burger,
+  plus,
+} from '../../../images'
 import WalletList from './WalletList'
 
-const mapStateToProps = (ownState, ownProps) => {
-  return {
-    
+const sectionsSelector = () => { }
+const sections = [
```

Comment:
Such test data should be placed into `src/screens/Wallet/WalletList/testData.json` file and imported in this file.
That will provide us with clear sign that here we have stubbed data. 

Any data like this should be placed in `testData.json` (exactly this name in different folders for different components).